tgui/packages/tgui/interfaces/NtosDeptOrder.tsx

```diff
@@ -44,12 +46,12 @@ type Info = {
 const CooldownEstimate = (props) => {
   const { cost } = props;
   const cooldownColor =
-    (cost > COST_UPPER_BOUND * 0.75 && 'red') ||
-    (cost > COST_UPPER_BOUND * 0.25 && 'orange') ||
+    (cost >= COST_UPPER_BOUND && 'red') ||
+    (cost >= COST_MEDIUM_BOUND && 'orange') ||
     'green';
   const cooldownText =
-    (cost > COST_UPPER_BOUND * 0.75 && 'long') ||
-    (cost > COST_UPPER_BOUND * 0.25 && 'moderate') ||
+    (cost >= COST_UPPER_BOUND && 'long') ||
+    (cost >= COST_MEDIUM_BOUND && 'moderate') ||
```

Comment:
I would consider tweaking the way we're representing the time on the console. As an example, ordering a BZ canister versus a recovered employee is a 12 minute versus 18 minute cost at current. However, the employee is labeled as "short" while the BZ is considered long.

Also you'll want to re-assess this following the other review's comments on tweaking the cooldown formula

Reply:
Altered.